views/about.lavender

```diff
@@ -24,7 +24,7 @@ body
       .navbar-collapse.collapse
         ul.nav.navbar-nav
           li
-            a(href="/lavender.html", title="Home") Home
+            a(href="/index.html", title="Home") Home
```

Comment:
Those links to anchors within the page are working properly for me. Putting a slash before the file makes my browser look for the file in the root (file:///index.html) instead of relatively (based on where the current file is located, file:///Users/danmowchan/golavender.github.io/index.html).